src/rct1/S4Importer.cpp

```diff
@@ -979,8 +979,8 @@ void S4Importer::ImportParkFlags()
     }
 
     // News items
-    rct_news_item *newsItems = RCT2_ADDRESS(RCT2_ADDRESS_NEWS_ITEM_LIST, rct_news_item);
-    for (int i = 0; i < 61; i++)
+    rct_news_item *newsItems = gNewsItems;
+    for (int i = 0; i < MAX_NEWS_ITEMS; i++)
```

Comment:
This is one less than before (presumably the array was one bigger than you have currently got), I don't know what is correct.